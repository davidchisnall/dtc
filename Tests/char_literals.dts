/dts-v1/;

/ {

    gpio_port = <('F' - 'A')>;
    test_nl   = <('\n')>;
    test_cr   = <('\r')>;
    test_tab  = <('\t')>;
    test_bs   = <('\\')>;
    test_sq   = <('\'')>;
    test_zero = <('\0')>;

}
