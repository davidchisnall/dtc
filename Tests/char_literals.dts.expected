/dts-v1/;

/  {

	gpio_port = <0x5>;
	test_nl = <0xa>;
	test_cr = <0xd>;
	test_tab = <0x9>;
	test_bs = <0x5c>;
	test_sq = <0x27>;
	test_zero = <0x0>;
};
